src/content/app/tools/blast/views/blast-submission-results/components/blast-results-per-sequence/BlastResultsPerSequence.scss

```diff
@@ -59,3 +59,13 @@
 .summaryPlot {
   grid-column: hits-diagram;
 }
+
+.collapsedJobStatus {
+  justify-self: end;
+  color: $red;
+  padding-right: 20px;
+}
+
+.failedJobStatus {
+  color: $red;
+}
```

Comment:
Is the job status always red? Would it make sense to have a `.jobStatus` (or `.faliedJob`) class that says that its color is red, and then either a `.collapsedJobStatus` class or (even better) a `.resultsSummaryRow .jobStatus` combo that sets `justify-self` and `padding-right`?

Reply:
Confused, lets discuss this over code review